packages/core/src/bodl/bodl-emitter-service.ts

```diff
@@ -75,7 +77,9 @@ export default class BodlEmitterService implements BodlService {
             shippingCostTotal,
             lineItems,
             cartId,
-            coupons
+            coupons,
+            channelId,
```

Comment:
From what I can see we don't have `channelId` here, can you please add it here as well
https://github.com/bigcommerce/checkout-sdk-js/blob/master/packages/core/src/order/order.ts#L9

Reply:
Same as above